extensions/amp-mustache/0.1/test/test-amp-mustache.js

```diff
@@ -424,6 +486,49 @@ describe('amp-mustache 0.1', () => {
     });
   });
 
+  describe('tables', () => {
+    it('should not foster text nodes in script template', done => {
+      if (isTemplateTypeScript) {
+        textContentSetup(
+            '<table>' +
+              '<tbody>' +
+                '<tr>' +
+                  '<td>Comment:</td>' +
+                  '<td>{{content}}</td>' +
+                '</tr>' +
+                '{{#replies}}' +
+                '<tr>' +
+                  '<td>Reply:</td>' +
+                  '<td>{{content}}</td>' +
+                '</tr>' +
+              '{{/replies}}' +
+              '</tbody>' +
+            '</table>');
+        template.compileCallback();
+        allowConsoleError(() => {
+          const result = template.render({
+            'content': 'Howdy',
+            'replies': [{'content': 'hi'}],
+          });
+          expect(result.innerHTML).to.equal(
+              '<tbody>' +
+                '<tr>' +
+                  '<td>Comment:</td>' +
+                  '<td>Howdy</td>' +
+                '</tr>' +
+                '<tr>' +
+                  '<td>Reply:</td>' +
+                  '<td>hi</td>' +
+                '</tr>' +
+              '</tbody>');
+          done();
+        });
+      } else {
+        done();
+      }
+    });
+  });
```

Comment:
Ditto, except do the same for the script case.

Reply:
done.